packages/components/src/TCRUserDashboard/DashboardNewsroom.tsx

```diff
@@ -98,6 +104,11 @@ const DashboardNewsroomRegistryStatusBase: React.FunctionComponent<DashboardNews
 const DashboardNewsroomRegistryStatus = React.memo(DashboardNewsroomRegistryStatusBase);
 
 const DashboardNewsroomBase: React.FunctionComponent<DashboardNewsroomProps> = props => {
+  const [copied, setCopied] = React.useState(false);
+  const storyBoostEmbed = ReactDOMServer.renderToStaticMarkup(
+    <script src="http://registry.civil.co/loader/boost.js"></script>,
+  );
```

Comment:
I don't think there's any need to use `ReactDOMServer.renderToStaticMarkup` since there's no react code to render here, it's just plain HTML, so I'd store it as a string.